requirements.txt

```diff
@@ -10,10 +10,16 @@ python-dotenv==0.10.3
 gunicorn==20.0.4
 honcho==1.0.1
 
-# Testing
+# TDD testing
 nose==1.3.7
 rednose==1.3.0
 pinocchio==0.4.2
 factory-boy==2.12.0
 coverage==4.5.4
 pylint>=2.8.3
+
+# BDD testing
+behave==1.2.6
+selenium==3.141.0
+compare==0.2b0
+requests==2.25.1
```

Comment:
Hey, these lines were supposed to be part of issue #43 - do you mind removing them here, or editing that ticket please?

Reply:
I just updated #43 : )